### PR TITLE
Disable snmpd module disk_hw, so no syslog messages about unavailable disks

### PR DESCRIPTION
**Pull request by qiluo-msft**
**- What I did**
Fixed https://github.com/Azure/sonic-snmpagent/issues/22

**- How I did it**
Disabled the module of snmpd

**- How to verify it**
Tested in lab

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: dockers/docker-snmp-sv2/snmpd.conf.j2 [hunk 1]
```diff
@@ -119,7 +114,7 @@ load   12 10 5
 #  Event MIB - automatically generate alerts
 #
 # Remember to activate the 'createUser' lines above
-#iquerySecName   internalUser
+#iquerySecName   internalUser
```
Comment (stcheng) [inline dockers/docker-snmp-sv2/snmpd.conf.j2]: trailing spaces.

File: dockers/docker-snmp-sv2/snmpd.conf.j2 [hunk 2]
```diff
@@ -61,7 +61,7 @@ sysServices    72
 #
 #  Process Monitoring
 #
-# TODO: should we enable snmp based monitoring of sswsyncd and other processes?
+# todo: should we enable snmp based monitoring of sswsyncd and other processes?
```
Comment (jleveque) [inline dockers/docker-snmp-sv2/snmpd.conf.j2]: Why change TODO to lowercase? All other TODOs in the file are uppercase.

Comment (qiluo-msft) [reply dockers/docker-snmp-sv2/snmpd.conf.j2]: It's in a revert commit. Since it is a comment format issue, I will not refix it again.

**Comment by hzheng5** [comment]
@qiluo-msft Is this issue fixed? 

We have the snmpd module disk_hw disabled(commit 3e2b45aa6be3453ed117f3cc611f56bcdef6b81f). However, we are still seeing the following error msgs:

Jul 31 01:50:05.066931 PSW-APPDB-G3-P14-1.na61 ERR snmpd[35]: Cannot statfs /root/host: Permission denied
Jul 31 01:50:05.066931 PSW-APPDB-G3-P14-1.na61 ERR snmpd[35]: Cannot statfs /root/dev: Permission denied
Jul 31 01:50:05.066931 PSW-APPDB-G3-P14-1.na61 ERR snmpd[35]: Cannot statfs /root/dev/pts: Permission denied

Thanks,
Haiyang

**Comment by qiluo-msft** [comment]
We did one more fix https://github.com/Azure/sonic-buildimage/pull/1668

**Comment by hzheng5** [comment]
I see. Thanks Qi Luo!

**Pull request merged**